From 89b2c24e30e6da11abea4f604e3f5ce224ad8992 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Tue, 1 Jul 2014 18:18:38 +0300
Subject: [PATCH 1/2] HummingBoard - Fix for faking USB OTG host on solo /
 dual lite

The pin muxing for HummingBoard was buggy and this fix adds correct USB OTG ID
faking mehanism via unconnected RX_ER pin.

The result is that the USB OTG phy will detect the USB interface as host
interface as required (vs. device).

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 arch/arm/mach-mx6/board-mx6dl_hb.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6dl_hb.h b/arch/arm/mach-mx6/board-mx6dl_hb.h
index 46db061..eb10922 100644
--- a/arch/arm/mach-mx6/board-mx6dl_hb.h
+++ b/arch/arm/mach-mx6/board-mx6dl_hb.h
@@ -71,7 +71,7 @@ static iomux_v3_cfg_t mx6dl_hb_pads[] = {
 	 * A better way to do this is by chaning the driver itself to accept
 	 * a flag indicating that this USB OTG will work only in host mode.
 	 */
-	IOMUX_PAD(0x05C4, 0x01F4, 0, 0x0790, 1, PAD_CTL_PKE | PAD_CTL_PUE | \
+	IOMUX_PAD(0x05C4, 0x01F4, 0, 0x0790, 0, PAD_CTL_PKE | PAD_CTL_PUE | \
 		PAD_CTL_PUS_100K_DOWN | PAD_CTL_SPEED_LOW |	\
 		PAD_CTL_DSE_80ohm | PAD_CTL_SRE_FAST | PAD_CTL_HYS),
 
-- 
1.7.9.5

